src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -191,17 +205,18 @@ public static AbstractBuilding createFromNBT(Colony colony, NBTTagCompound compo
             try
             {
                 building.readFromNBT(compound);
-            }
-            catch (RuntimeException ex)
+            } catch (RuntimeException ex)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "catch" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)